src/components/text-search/_autocomplete.scss

```diff
@@ -0,0 +1,140 @@
+/**
+ * autocomplete.scss
+ * Define autocomplete input + suggestion styles.
+ * @see: https://github.com/moroshko/react-autosuggest#theme-optional
+ *
+ * @copyright Copyright (c) 2017 Palantir.net
```

Comment:
Needs date update.

Reply:
Thanks. Updated in https://github.com/palantirnet/federated-search-react/pull/26/commits/85fcd96a25e5fde0b32e8a17f4b4813bc222e25f